### PR TITLE
add courses of sass, logic and bootstrap

### PR DESCRIPTION
**Pull request by priscillatrevizan**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -374,6 +381,7 @@
 
 ### Sass
 
+* [Curso de Sass](https://www.youtube.com/playlist?list=PL97KElaimHeGRtfkksKwxg6IGVZi_cR7J) - Amanda Vilela (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:384]:
```suggestion
* [Curso de Sass](https://www.youtube.com/playlist?list=PL97KElaimHeGRtfkksKwxg6IGVZi_cR7J) - Amanda Vilela
```

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -254,6 +256,11 @@
 * [Terminal Linux](https://www.udemy.com/course/terminal-de-comandos-linux/) - Diego Mariano (Udemy)
 
 
+### Lógica de Programação
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:259]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -162,6 +163,7 @@
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 2 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dlUpEXkY1AyVLQGcpSgVF8s) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 3 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dmcAviDwiGgHbeEJToxbOpZ) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 4 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dkcVCk2Bn_fdVQ81Fkrh6WT) - Gustavo Guanabara (Curso em Vídeo)
+* [Curso de HTML, CSS e Bootstrap](https://www.youtube.com/playlist?list=PL8iIphQOyG-Cv3auRYoZtbvzJJrsvdMy-) - Michelli Brito (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:166]:
```suggestion
* [Curso de HTML, CSS e Bootstrap](https://www.youtube.com/playlist?list=PL8iIphQOyG-Cv3auRYoZtbvzJJrsvdMy-) - Michelli Brito
```
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Review by eshellman — CHANGES_REQUESTED** [review]
Programming logic? Could that be Algorithms?

**Comment by priscillatrevizan** [comment]
@eshellman 
changes made, ready for code review

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -162,6 +162,12 @@
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 2 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dlUpEXkY1AyVLQGcpSgVF8s) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 3 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dmcAviDwiGgHbeEJToxbOpZ) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 4 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dkcVCk2Bn_fdVQ81Fkrh6WT) - Gustavo Guanabara (Curso em Vídeo)
+* [Curso de HTML, CSS e Bootstrap - Aula 1 de 6 ](https://www.youtube.com/watch?v=QZ8mu8vuOpg&list=PL8iIphQOyG-Cv3auRYoZtbvzJJrsvdMy-&index=1) - Michelli Brito (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:165]: is there no playlist we can use for this?
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
Maybe we should find a different link for the Gustavo Guanabara modules.

**Pull request closed without merging**